### PR TITLE
Disable controls if the candidate is marked as hired or disqualified.

### PR DESCRIPTION
**Pull request by jscottchapman**
Added the ability to undo the hiring action in the case of someone
accidentally hitting the hire button on the wrong person.

### CODE REVIEW

File: app/components/card-pipeline/stage-card.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ const PipelineCardComponent = Component.extend({
     return this.get('getNextMeeting').perform();
   }),
 
+  controlsDisabled: computed('application.hired', 'application.disqualified', function () {
```
Comment (James1x0) [inline app/components/card-pipeline/stage-card.js]: computed path could be `application.{hired,disqualified}`

File: app/controllers/account/job-opening/campaign/applicant-tracking.js [hunk 2]
```diff
@@ -174,6 +174,11 @@ export default Controller.extend(addEdit, ajaxStatus, modalSupport, {
     unDisqualifyCandidate (jobApp) {
       jobApp.set('disqualified', false);
       this.saveModel(jobApp);
+    },
+
+    unSetHire (jobApp) {
```
Comment (James1x0) [inline app/controllers/account/job-opening/campaign/applicant-tracking.js]: More will probably have to be done to make "unhiring" work correctly. We will have to roll back title changes or delete employee records for applicants to make it _feel valid_, right?

Comment (jscottchapman) [reply app/controllers/account/job-opening/campaign/applicant-tracking.js]: @James1x0 So, should I remove the unSetHire for now or leave it in there?

Comment (James1x0) [reply app/controllers/account/job-opening/campaign/applicant-tracking.js]: Let's remove dead code at the moment.

**Pull request merged**